libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/DefaultSpanScope.java

```diff
@@ -44,23 +44,23 @@ private DefaultSpanScope(Span span, SpanScope previousSpanScope, TracerContextSt
     public static SpanScope create(Span span, TracerContextStorage<String, Span> tracerContextStorage) {
         final SpanScope beforeSpanScope = spanScopeThreadLocal.get();
         SpanScope newSpanScope = new DefaultSpanScope(span, beforeSpanScope, tracerContextStorage);
-        spanScopeThreadLocal.set(newSpanScope);
```

Comment:
So create no longer attaches?

Reply:
Nope and it shouldn't, the `attach` does